### PR TITLE
Parameterized Dockerfile values

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Parameterize Dockerfile values.  Moved RHEL_VERSION to a build argument

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
790ea934-5b6e-4032-97ea-6c71fef5b0a7

File: test-container/Dockerfile [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
+ARG RHEL_VERSION=8
+FROM registry.access.redhat.com/ubi${RHEL_VERSION}/ubi AS builder
 # Download specific version of staticcheck
 RUN curl -L https://github.com/dominikh/go-tools/releases/download/2021.1.2/staticcheck_linux_amd64.tar.gz | tar xz
 # Download specific version of gosec
 RUN curl -L https://github.com/securego/gosec/releases/download/v2.9.5/gosec_2.9.5_linux_amd64.tar.gz | tar xz
 
-# Install RHEL 8 utilities to /mnt/rootfs
-RUN yum install --installroot /mnt/rootfs \
+# Install RHEL utilities to install root
```
Comment (loadtheaccumulator) [inline test-container/Dockerfile:18]: I had to read this one twice, but it's early here. :-)
Maybe "Install RHEL utilities into install root" ???

Comment (thom-at-redhat) [reply test-container/Dockerfile]: thanks to your help, improved comment

**Review by loadtheaccumulator — APPROVED** [review]
lgtm

**Pull request merged**